src/tlo/methods/healthsystem.py

```diff
@@ -379,6 +387,12 @@ def __init__(
         if mode_appt_constraints is not None:
             assert mode_appt_constraints in {0, 1, 2}
         self.arg_mode_appt_constraints = mode_appt_constraints
+        
+        self.year_mode_switch = None
+        self.arg_year_mode_switch = year_mode_switch
+        
+        self.scale_to_effective_capabilities = None
+        self.arg_scale_to_effective_capabilities = scale_to_effective_capabilities
```

Comment:
```suggestion
        self._arg_year_mode_switch = year_mode_switch
        self._arg_scale_to_effective_capabilities = scale_to_effective_capabilities
```
The `self.arg_*` attributes seem to be only for internal use so should probably be prefixed with an underscore to hint this and there is no need to separately initialise the `self.scale_to_effective_capabilities` and `self.year_mode_switch` attributes if changing to properties as suggested below.